src/navigation/components/NavigationItem/components/NavigationDropdown/NavigationDropdown.tsx

```diff
@@ -16,7 +16,7 @@ const TOGGLE_ARROW_SIZE = 12;
 type NavigationDropdownProps = NavigationItemProps & DropdownItemData;
 
 export const NavigationDropdown = React.forwardRef<HTMLElement, NavigationDropdownProps>(
-    ({text, icon, isOpened, className, iconSize, ...props}, ref) => {
+    ({text, icon, isOpened, className, iconSize, type: _, ...props}, ref) => {
```

Comment:
??

Reply:
Attribute type not allowed on element [span](https://html.spec.whatwg.org/multipage/#the-span-element) at this point. span tag does not have 'type' attribute and validator.w3.org informs us about this. So I cannot see technical reason to set it

Reply:
Is `type` used anywhere?
Maybe it can be `div` instead of `span`?

Reply:
how I see we don't read elements to get his attribute. 'Type' is used somewhere upper in component. I have decided not to change to div, because for my case it does not matter span or div in this component.